components/scripts/lib/info.sh

```diff
@@ -230,3 +263,21 @@ create_receipt_file() {
   echo "Generated by $(print_version)"
   } > "${RECEIPT_FILE}"
 }
+
+format_duration() {
```

Comment:
You may be wondering why I'm formatting the duration here when it's already done in the Java tool. It's a good question. 🙂 

The reason is because in order to calculate the realized build time savings, I need to compute the difference in the effective task duration between the two builds. Prior to this change, there were no metrics that required a property from both builds.

Since the command line tools (both offline & online) process Build Scan data independently, I needed the effective task execution duration to be returned to the build validation scripts as a numerical value (as milliseconds), rather than as a pre-formatted duration. With the numerical value, I can then compute the realized build time savings within the build validation scripts. 

This introduces an inconsistency in the format of the data returned by the command line tools. Some durations are returned pre-formatted and this field is not.

Therefore, I would like to propose we remove formatting logic from the command line tools and always return durations as a milliseconds. We could then remove the duplicated formatting logic in the command line tools.

Reply:
It would be good to capture this comment in one of the commit messages, if possible.